<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>学生管理系统</string>
  </property>
  <property name="toolTip">
   <string>这是学生管理系统</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget#centralwidget(background-color: rgb(0, 0, 255););</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QWidget" name="widget" native="true">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(85, 0, 255);
color: rgb(255, 255, 255);</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1,1,1">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>Agency FB</family>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="text">
          <string>学生管理系统</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>admin</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 170, 0);</string>
         </property>
         <property name="text">
          <string>退出</string>
         </property>
         <property name="autoRepeatDelay">
          <number>303</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QTreeWidget" name="treeWidget">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>160</width>
        <height>16777215</height>
       </size>
      </property>
      <attribute name="headerVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="6">
         <widget class="QPushButton" name="pushButton_5">
          <property name="text">
           <string>删除</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QCheckBox" name="checkBox">
          <property name="text">
           <string>全选</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="btn_search">
          <property name="text">
           <string>搜索</string>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QPushButton" name="pushButton_4">
          <property name="text">
           <string>修改</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="7">
         <widget class="QTableWidget" name="tableWidget">
          <row>
           <property name="text">
            <string/>
           </property>
          </row>
          <column>
           <property name="text">
            <string>序号</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>姓名</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>年龄</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>年级</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>班级</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>学号</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>微信</string>
           </property>
          </column>
          <item row="0" column="0">
           <property name="text">
            <string>1</string>
           </property>
          </item>
          <item row="0" column="1">
           <property name="text">
            <string>小明</string>
           </property>
          </item>
          <item row="0" column="2">
           <property name="text">
            <string>20</string>
           </property>
          </item>
          <item row="0" column="3">
           <property name="text">
            <string>1</string>
           </property>
          </item>
          <item row="0" column="4">
           <property name="text">
            <string>1</string>
           </property>
          </item>
          <item row="0" column="5">
           <property name="text">
            <string>201</string>
           </property>
          </item>
          <item row="0" column="6">
           <property name="text">
            <string>xiaoming</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QPushButton" name="pushButton_add">
          <property name="text">
           <string>增加</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="btn_moni">
          <property name="text">
           <string>模拟数据</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="btn_clear">
          <property name="text">
           <string>清空学生表</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="2">
         <widget class="QLineEdit" name="le_search"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
